src/main/java/org/spongepowered/common/mixin/core/network/MixinNetHandlerPlayServer.java

```diff
@@ -347,5 +350,32 @@ public void onDisconnectPlayer(IChatComponent reason, CallbackInfo ci) {
         Sponge.getGame().getEventManager().post(event);
         event.getSink().sendMessage(event.getNewMessage());
     }
+    
+    @Inject(method = "processUseEntity", at = @At(value = "INVOKE", target =
+             "net/minecraft/network/play/client/C02PacketUseEntity.getAction()Lnet/minecraft/network/play/client/C02PacketUseEntity$Action;",
+             ordinal = 0), cancellable = true, locals = LocalCapture.CAPTURE_FAILSOFT)
+    public void processUseEntity(C02PacketUseEntity packetIn, CallbackInfo info, WorldServer worldserver, Entity entity, boolean flag, double d0) {
```

Comment:
I believe the `boolean flag` parameter needs the `@Coerce` annotation. `double` is fine I think